extensions/amp-analytics/0.1/variables.js

```diff
@@ -277,14 +277,12 @@ export class VariableService {
       let value = options.getVar(name);
 
       if (typeof value == 'string') {
-        value = this.expandTemplateSync(
-          value,
-          new ExpansionOptions(
-            options.vars,
-            options.iterations - 1,
-            true /* noEncode */
-          )
-        );
+        value = this.expandValue_(value, options);
+      } else if (isArray(value)) {
+        // Treat each value as a template and expand
+        for (let i = 0; i < value.length; i++) {
```

Comment:
The code breaks with an array like `["test", 123]`, because we are checking if the value type of string for items from the array. 

This breaks use case like:
```
"request": "example.com?data=${data1}"    
"vars": {
      "abc": "${timestamp}.${requestCount}",
      "data1": ["ds", 123]
    },
``` 
Let's fix before next canary cut, thanks.